events/HF_battle_events.txt

```diff
@@ -740,6 +740,54 @@ character_event = {
 				}
 				character_event = { id = HFP.30200 }
 			}
+
+            # Warcraft
+			30 = {
+				trigger = {
+					is_alive = yes
+					
+					can_have_more_leadership_traits = yes
+					NOT = { has_character_modifier = battle_events_7_cooldown } #all commander traits-giving events are set to need a year in between firing...
+				}
+				good_soldier_outcome_score = yes
+
+				add_character_modifier = { name = battle_events_7_cooldown duration = 365 hidden = yes }
+				disallow_new_battle_events_effect = yes
+				battle_commander_counter_positive_effect = yes
```

Comment:
are you sure that warcraft event should check vanilla crusades in order to check if character deserve crusader commander bloodline?